packages/web3-utils/src/converters.ts

```diff
@@ -0,0 +1,340 @@
+import { BigDecimal } from './big_decimal';
+import { sha3 } from './hash';
+import { Address, Bytes, HexString, Numbers, ValueTypes } from './types';
+import { isAddress, isHexStrict } from './validation';
+
+const base = BigInt(10);
+const expo10 = (expo: number) => base ** BigInt(expo);
+
+// Ref: https://ethdocs.org/en/latest/ether.html
+/** @internal */
+const ethUnitMap = {
+	noether: BigInt('0'),
+	wei: BigInt(1),
+	kwei: expo10(3),
+	Kwei: expo10(3),
+	babbage: expo10(3),
+	femtoether: expo10(3),
+	mwei: expo10(6),
+	Mwei: expo10(6),
+	lovelace: expo10(6),
+	picoether: expo10(6),
+	gwei: expo10(9),
+	Gwei: expo10(9),
+	shannon: expo10(9),
+	nanoether: expo10(9),
+	nano: expo10(9),
+	szabo: expo10(12),
+	microether: expo10(12),
+	micro: expo10(12),
+	finney: expo10(15),
+	milliether: expo10(15),
+	milli: expo10(15),
+	ether: expo10(18),
+	kether: expo10(21),
+	grand: expo10(21),
+	mether: expo10(24),
+	gether: expo10(27),
+	tether: expo10(30),
+};
+
+const errorMessage = (value: unknown, error: string) =>
```

Comment:
I would think this should be defined in `errors.ts` to be used by all the inline errors below